app/src/main/res/layout/item_search_view.xml

```diff
@@ -35,6 +35,7 @@
         <com.google.android.material.textfield.TextInputEditText
             android:id="@+id/searchTextInput"
             style="@style/SearchTextInputEditTextStyle"
+            android:backgroundTint="@android:color/transparent"
```

Comment:
Is it related to this feature ?

Reply:
Yeah, the background color of TextInputEditText changed for this feature and this line is needed to keep the same look with the new changes